all_test.go

```diff
@@ -159,6 +163,59 @@ func hasValidHeader(path string, r io.Reader) (bool, error) {
 	return headerRegex.Match(allBytes), nil
 }
 
+func TestConsistentGoVersions(t *testing.T) {
```

Comment:
Add a comment about why we have this test. We should consider just moving the failing test to presubmit, in which case I don't think we will need this test. It's fine to merge this PR for now though.